internal/tools/securitypolicy/helpers/helpers.go

```diff
@@ -255,3 +256,49 @@ func setDefaultUser(config *sp.UserConfig, user, group sp.IDNameConfig) sp.UserC
 		Umask:        "0022",
 	}
 }
+
+func setDefaultCapabilities(config sp.CapabilitiesConfig, allowElevated bool) sp.CapabilitiesConfig {
+	// From the toml marshaller, if there was no entry then it will be `nil`.
+	// We should provide defaults if it was missing
+	if config.Bounding == nil {
+		if allowElevated {
+			config.Bounding = sp.DefaultPrivilegedCapabilities()
+		} else {
+			config.Bounding = sp.DefaultUnprivilegedCapabilities()
+		}
+	}
+
+	if config.Effective == nil {
+		if allowElevated {
+			config.Effective = sp.DefaultPrivilegedCapabilities()
+		} else {
+			config.Effective = sp.DefaultUnprivilegedCapabilities()
+		}
+	}
+
+	if config.Inheritable == nil {
+		if allowElevated {
+			config.Inheritable = sp.DefaultPrivilegedCapabilities()
+		} else {
+			config.Inheritable = sp.EmptyCapabiltiesSet()
```

Comment:
@SeanTAllen this is unexpected. Is this another thing with weird defaults being set in containerd?

Reply:
yes, inheritable is also empty by default.